Tools/install.sh

```diff
@@ -22,6 +22,46 @@ FONT_PACKAGES=(\
     fonts-droid-fallback \
     )
 
+check_for_installed_tools () {
```

Comment:
I'm not keen on the location of these two functions as they split the declaration of `FONT_PACKAGES` from the `EXTRA_PACKAGES` and `OPTIONAL_PACKAGES`. Maybe move them to the end of the script?

Reply:
End of script does not work, but I moved the code down in the file.